gulpfile.js

```diff
@@ -23,15 +23,15 @@ gulp.task('browser', function() {
     .pipe(gulp.dest('./Example/source'));
 });
 
-gulp.task('es6', function(){
+gulp.task('es6',_ => {
```

Comment:
Hey, I just checked this PR locally. Your gulp file isn't finishing process. Could you please check?